configs/bluenove-server-configs/etalab-snapshot.rc

```diff
@@ -0,0 +1,7 @@
+#In order to get credentials for your script, log into https://api.ovh.com/createToken
```

Comment:
This file should not be created in this repository but in the other one

Reply:
And probably these new keys should be in a category like `[ovh_hosting]` and documented in `doc/configuration.rst`

Reply:
yeah good idea ... probably should be documented in doc/configuration.rst ... but what do you mean by in a new category [ovh_hosting] ?

Reply:
I mean that when you open local.ini, you see that variables are into groups that are shown like [this]

Reply:
And in an .rc file, when you define a variable that is in a group, the variable name starts with the group name, then double underscore, then the variable name. Example: `theme_repositories__git-urls`